db/migrate/20230109165658_drop_user_mfa_required_prompt_skip_date.rb

```diff
@@ -0,0 +1,5 @@
+class DropUserMfaRequiredPromptSkipDate < ActiveRecord::Migration[7.0]
```

Comment:
There should be a change to the `db/schema.rb` `version` as well, right? I noticed I was never prompted to do the migration, which I think is related.

Reply:
Separately, I wonder if we should have some checks for that, since I've run into that before myself where the default migration generator outputs to the wrong directory. Seems easy to overlook.